packages/cli/package.json

```diff
@@ -56,7 +56,9 @@
     "@prisma/cli": "2.0.0-beta.3",
     "@rollup/pluginutils": "^3.0.8",
     "@types/pluralize": "^0.0.29",
-    "nock": "13.0.0-beta.3"
+    "@types/rimraf": "^3.0.0",
+    "nock": "13.0.0-beta.3",
+    "rimraf": "^3.0.2"
```

Comment:
Can you move these to the root package.json? (rimraf is already installed there, btw). Then general rule is to install all dev dependencies in the project root.